src/docs/pages/home.vue

```diff
@@ -28,6 +28,13 @@ export default defineComponent({
         this.validCard = card
       }
     })
+
+    var main = this.$refs.example as HTMLDivElement
+    var newChild = document.createElement('div')
+    newChild.id = 'latediv'
+    setTimeout(() => {
+      main.appendChild(newChild)
+    }, 100)
```

Comment:
I tested with 10000ms delay and got the following error on the console:
```
index.ts?144e:87 Uncaught Error: Could not find container
    at eval (index.ts?144e:87:1)
    at eval (index.ts?144e:186:1)
```

Reply:
Thats a good catch. But its intentional. It will stop attempting to look for the container after 10,000ms. Otherwise it would just keep looking. I figured if tokenizer is still looking for a container after 10,000ms the container doesnt exist.